__tests__/data/tiktok_ads_output.json

```diff
@@ -1234,5 +1234,214 @@
+      },
+      "files": {}
+    }
+  ],
+  [
+    {
+      "version": "1",
+      "type": "REST",
+      "method": "POST",
+      "endpoint": "https://business-api.tiktok.com/open_api/v1.2/pixel/track/",
+      "headers": {
+        "Access-Token": "1234ac663758946dfea3520b394bbac511b371f7",
+        "Content-Type": "application/json"
+      },
+      "params": {},
+      "body": {
+        "JSON": {
+          "pixel_code": "A1T8T4UYGVIQA8ORZMX9",
+          "partner_name": "RudderStack",
+          "event": "CompletePayment",
+          "event_id": "1616318632825_357",
+          "timestamp": "2020-09-17T19:49:27Z",
+          "properties": {
+            "contents": [
+              {
+                "content_id": "123",
+                "content_type": "product",
+                "content_name": "Monopoly",
+                "price": 14,
+                "quantity": 1,
+                "content_category": "Games"
+              },
+              {
+                "content_id": "345",
+                "content_type": "product",
+                "content_name": "UNO",
+                "price": 3.45,
+                "quantity": 2
+              }
+            ],
+            "currency": "USD",
+            "value": 46
+          },
+          "context": {
+            "ad": {
+              "callback": "123ATXSfe"
+            },
+            "page": {
+              "url": "http://demo.mywebsite.com/purchase",
+              "referrer": "http://demo.mywebsite.com"
+            },
+            "user": {
+              "external_id": "f0e388f53921a51f0bb0fc8a2944109ec188b59172935d8f23020b1614cc44bc",
+              "phone_number": "2f9d2b4df907e5c9a7b3434351b55700167b998a83dc479b825096486ffcf4ea",
+              "email": "dd6ff77f54e2106661089bae4d40cdb600979bf7edc9eb65c0942ba55c7c2d7f"
+            },
+            "ip": "13.57.97.131",
+            "user_agent": "Mozilla/5.0 (platform; rv:geckoversion) Gecko/geckotrail Firefox/firefoxversion"
+          }
+        },
+        "JSON_ARRAY": {},
+        "XML": {},
+        "FORM": {}
+      },
+      "files": {}
+    }
+  ],
+  [
+    {
+      "version": "1",
+      "type": "REST",
+      "method": "POST",
+      "endpoint": "https://business-api.tiktok.com/open_api/v1.2/pixel/track/",
+      "headers": {
+        "Access-Token": "1234ac663758946dfea3520b394bbac511b371f7",
+        "Content-Type": "application/json"
+      },
+      "params": {},
+      "body": {
+        "JSON": {
+          "pixel_code": "A1T8T4UYGVIQA8ORZMX9",
+          "partner_name": "RudderStack",
+          "event": "CompletePayment",
+          "event_id": "1616318632825_357",
+          "timestamp": "2020-09-17T19:49:27Z",
+          "properties": {
+            "contents": [
+              {
+                "price": 8,
+                "quantity": 2,
+                "content_type": "socks",
+                "content_id": "1077218"
+              },
+              {
+                "content_id": "123",
```

Comment:
Can we validate whether all these 6 properties are supported by tiktok - Are they getting displayed in dashboard ?